pages/treeselect/index.js

```diff
@@ -37,19 +59,76 @@ const TreeSelectDemo = () => {
+                        placeholder="Select Item"
+                    ></TreeSelect>
 
                     <h5>Multiple</h5>
-                    <TreeSelect value={selectedNodeKeys1} options={nodes} onChange={(e) => setSelectedNodeKeys1(e.value)} selectionMode="multiple" metaKeySelection={false} placeholder="Select Items"></TreeSelect>
+                    <TreeSelect
+                        value={selectedNodeKeys1}
+                        expandedKeys={expandedKeys}
+                        options={nodes}
+                        onToggle={(e) => setExpandedKeys(e.value)}
+                        onChange={(e) => setSelectedNodeKeys1(e.value)}
+                        selectionMode="multiple"
+                        metaKeySelection={false}
+                        placeholder="Select Items"
+                    ></TreeSelect>
 
                     <h5>Checkbox</h5>
-                    <TreeSelect value={selectedNodeKeys2} options={nodes} onChange={(e) => setSelectedNodeKeys2(e.value)} display="chip" selectionMode="checkbox" placeholder="Select Items"></TreeSelect>
+                    <TreeSelect
+                        value={selectedNodeKeys2}
+                        expandedKeys={expandedKeys}
+                        options={nodes}
+                        onToggle={(e) => setExpandedKeys(e.value)}
+                        onChange={(e) => setSelectedNodeKeys2(e.value)}
+                        display="chip"
+                        selectionMode="checkbox"
+                        placeholder="Select Items"
+                    ></TreeSelect>
 
                     <h5>Filter</h5>
-                    <TreeSelect value={selectedNodeKey2} options={nodes} onChange={(e) => setSelectedNodeKey2(e.value)} filter placeholder="Select Items"></TreeSelect>
+                    <TreeSelect
+                        value={selectedNodeKey2}
+                        expandedKeys={expandedKeys}
+                        options={nodes}
+                        onToggle={(e) => setExpandedKeys(e.value)}
+                        onChange={(e) => setSelectedNodeKey2(e.value)}
+                        filter
+                        placeholder="Select Items"
+                    ></TreeSelect>
 
                     <h5>Initial Value</h5>
-                    <TreeSelect value={selectedNodeKey3} options={nodes} onChange={(e) => setSelectedNodeKey3(e.value)} placeholder="Select Item"></TreeSelect>
+                    <TreeSelect
+                        value={selectedNodeKey3}
+                        expandedKeys={expandedKeys}
+                        options={nodes}
+                        onToggle={(e) => setExpandedKeys(e.value)}
+                        onChange={(e) => setSelectedNodeKey3(e.value)}
+                        placeholder="Select Item"
+                    ></TreeSelect>
+
+                    <h5>Programmatic Control</h5>
+                    <div className="mb-4">
+                        <Button
```

Comment:
Shoudln';t for this Programmatic Control Demo we only use 1 TreeSelect and use a new TreeSelect. I know Mert likes exact examples and not updating the previous examples?